kof-operator/go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/grafana/grafana-operator/v5 v5.16.0
 	github.com/onsi/ginkgo/v2 v2.23.4
 	github.com/onsi/gomega v1.37.0
-	golang.org/x/net v0.40.0 // indirect; https://github.com/k0rdent/kof/security/dependabot/13
+	golang.org/x/net v0.39.0 // indirect; https://github.com/k0rdent/kof/security/dependabot/13
```

Comment:
Looks like some revert again, please update golang to the version at least '1.24.3' to avoid this